app/configuration/config_loader.py

```diff
@@ -0,0 +1,125 @@
+
+        print(term.blue_on_black(f"\nLoading config: {expanded_path}"))
+
+        with open(expanded_path) as loaded_config:
+            loaded_config = yaml.load(loaded_config, Loader=yaml.FullLoader)
+
+        for key, value in loaded_config.items():
+            self.config[key] = value
+            print(term.blue_on_black(f"Loaded {key}: {value}"))
+
+    def load_config(self, term: Terminal) -> Union[bool, dict]:
+        """Loads the configuration file."""
+        existing_paths = []
+
+        if isinstance(self.paths, str):
+            if os.path.isfile(self.paths):
+                existing_paths.append(self.paths)
+        elif isinstance(self.paths, tuple):
+            for path in self.paths:
+                if os.path.isfile(path):
+                    existing_paths.append(path)
+
+        if len(existing_paths) > 1:
+            print(
+                term.blue_on_black(
+                    "One or more configuration files were found to exist. "
+                    f"Using the one found at {existing_paths[0]}."
+                )
+            )
+
+        if (
+            not existing_paths
+            # The path is not in a default location,
+            # this means that it is None from an unrecognized system or was manually specified
+            and self.paths not in Configuration.configuration_paths.values()
+        ):
+            print(
+                term.red_on_black(
+                    "No configuration file found at that location or "
+                    "you are using a system with an unknown default configuration file location"
+                )
+            )
+            return False
+
+        elif (
+            # The path is in a default location, a path was probably not specified,
+            # unless the user pointed to one in the default location
+            not existing_paths
+            and self.paths in Configuration.configuration_paths.values()
+        ):
+            term.red_on_black(("No default configuration file found on your system"))
+            return False
+
+        # Choose the first path which is loaded
+        self.path_loaded = existing_paths[0]
+        self._load_yaml(term)  # Set config
+        return self.config
+
+    def config_loader_screen(self, term: Terminal) -> Union[dict, bool]:
+        """Screen showing the erros and configuration loaded by the module."""
+        print(term.home + term.clear + term.move_y(0))
+        print(term.bold_black_on_green(term.center("Loading Configurations")))
+        print("\n")
+
+        config = self.load_config(term)
+        if not config:
+            self.exit(term)
+            return False
+
+        # Configuration validator
+        response = fix_config(config, term)
+        if not response:
+            self.exit(term)
+            return False
+
+        print("\n\n")
+        print(
+            term.bold_black_on_green(
+                term.center("Successfully loaded configuration...")
```

Comment:
only show error screen no need to show anything if the Yaml got loaded correctly

Reply:
alright 👍🏾

Reply:
Fixed in 99541bcb0991efc050bc388dd1d0338cefaf7dd2.